tests/foreman/sys/test_fam.py

```diff
@@ -32,6 +41,48 @@ def sync_roles(target_sat):
         target_sat.cli.Ansible.roles_delete({'id': role_id})
 
 
+@pytest.fixture(scope='module')
+def setup_fam(module_target_sat, module_sca_manifest):
+    # Execute AAP WF for FAM setup
+    Broker().execute(workflow='fam-test-setup', source_vm=module_target_sat.name)
+
+    # Edit Makefile to not try to rebuild the collection when tests run
+    module_target_sat.execute(f"sed -i '/^live/ s/$(MANIFEST)//' {FAM_ROOT_DIR}/Makefile")
+
+    # Upload manifest to test playbooks directory
+    module_target_sat.put(f'{module_sca_manifest.path}', f'{module_sca_manifest.name}')
+    module_target_sat.execute(
+        f'mv {module_sca_manifest.name} {FAM_ROOT_DIR}/tests/test_playbooks/data'
+    )
+
+    # Edit config file
+    config_file = f'{FAM_ROOT_DIR}/tests/test_playbooks/vars/server.yml'
+    module_target_sat.execute(
+        f'cp {FAM_ROOT_DIR}/tests/test_playbooks/vars/server.yml.example {config_file}'
+    )
+    module_target_sat.execute(
+        f'sed -i "s/foreman.example.com/{module_target_sat.hostname}/g" {config_file}'
+    )
+    module_target_sat.execute(
+        f'sed -i "s/rhsm_pool_id:.*/rhsm_pool_id: {settings.subscription.rhn_poolid}/g" {config_file}'
+    )
+    module_target_sat.execute(
+        f'''sed -i 's/rhsm_username:.*/rhsm_username: "{settings.subscription.rhn_username}"/g' {config_file}'''
+    )
+    module_target_sat.execute(
+        f'''sed -i 's|subscription_manifest_path:.*|subscription_manifest_path: "data/{module_sca_manifest.name}.zip"|g' {config_file}'''
+    )
+    module_target_sat.execute(
+        f'''sed -i 's/rhsm_password:.*/rhsm_password: "{settings.subscription.rhn_password}"/g' {config_file}'''
+    )
```

Comment:
given the amount of changes we need to perform here, would it be easier/cleaner if we'd deploy this file (`server.yml`) from a template and completely ignore the example we ship in the package?

Reply:
No idea ;)
I was hoping there is just a way in robotello?

Reply:
I've found https://github.com/SatelliteQE/robottelo/blob/97d9117d74835665cad06b59d39ab5802a63b0ec/robottelo/hosts.py#L881-L892

Reply:
or https://github.com/SatelliteQE/robottelo/blob/97d9117d74835665cad06b59d39ab5802a63b0ec/robottelo/hosts.py#L1003-L1018

Reply:
We would still have to replace the hostname and manifest name during the test setup. For now I'd leave this as is even though it's ugly and we can change in the future.